projects/ngrx.io/content/guide/store/recipes/angular-elements.md

```diff
@@ -0,0 +1,113 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+The following recipe illustrates utilizing Angular Elements with a NgRx Store. This provides the ability to manage the state of a counter, and to select and display that counter state from within separate Angular Elements. This is all accomplished from within a single Angular Project and static HTML page.
+
+## Recipe
+
+1. Create a new application by running the below command
+```sh
+ng new ngRxElementDemo --createApplication=false
+```
+
+### Creating Library
+
+1. Create a new Angular Library using below command
+
+```sh
+ng g library my-counter
+```
+
+2. Follow the instructions in the [Getting Started Guide](guide/store#installation) to create a new counter store complete with actions and the reducer.
+3. For this recipe we have to split the component code in separate components, to showcase opportunities with Angular Elements. One component to increment the counter, one to decrement and another component to reset the counter.
+4. Rename `app.module.ts` to `counter.module.ts` file and add the below code.
+<code-example header="src/lib/counter.module.ts" path="store-elements/projects/my-counter/src/lib/counter.module.ts">
+</code-example>
+In the above code, we have added all the components in the `entryComponents` array. This ensures that the components are available for rendering as Angular Elements. In addition, inside the `constructor`, `createCustomElement` must be called for each component that will be used as an Angular Element.
+
+### Creating Applications
+
+1. Generate two (2) new applications within the project using the below command
+
+```sh
+ng g application elementApp
+ng g application counterelement
+```
+
+2. Run the below command to add Angular Element as a dependency to our project.
+
+```sh
+ng add @angular/elements
+```
+
+3. Once the command is completed, you will notice the below changes in `angular.json`, the above package only makes changes to the default project which in our case is `elementApp`, so copy the below line and add the same to the scripts section of `counterelement`.
+
+```json
+"scripts": [
+{"input": "node_modules/document-register-element/build/document-register-element.js"}
```

Comment:
```suggestion
  { "input": "node_modules/document-register-element/build/document-register-element.js "}
```

Reply:
HI @timdeschryver ,
Yes ngBootStrap is needed to build element. if we remove the elements will not work.